source/routing/tests/routing_api_test.cpp

```diff
@@ -46,7 +46,7 @@ www.navitia.io
 struct logger_initialized {
     logger_initialized()   { init_logger(); }
 };
-BOOST_GLOBAL_FIXTURE( logger_initialized );
+BOOST_GLOBAL_FIXTURE( logger_initialized )
```

Comment:
The semicolon generates a warning, but it is necessary when compiling with an older version of BOOST. It's ugly but it will prevent you from breaking the debian 7 build ;)